google-cloud-redis/google-cloud-redis.gemspec

```diff
@@ -0,0 +1,28 @@
+# -*- ruby -*-
+# encoding: utf-8
+
+Gem::Specification.new do |gem|
+  gem.name          = "google-cloud-redis"
+  gem.version       = "0.1.0"
+
+  gem.authors       = ["Google LLC"]
+  gem.email         = "googleapis-packages@google.com"
+  gem.description   = "google-cloud-redis is the official library for Google Cloud Memorystore for Redis API."
+  gem.summary       = "API Client library for Google Cloud Memorystore for Redis API"
+  gem.homepage      = "https://github.com/googleapis/googleapis"
```

Comment:
We should also update the homepage (example: https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/master/google-cloud-dlp/google-cloud-dlp.gemspec)

Reply:
I'll make this change.